src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,95 @@
+package storage;
+
+import tasks.*;
+
+import java.io.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Storage {
+    private final String filePath;
+    private final DateTimeFormatter inputFormat = DateTimeFormatter.ofPattern("MMM dd yyyy h:mm a");
+    private final DateTimeFormatter outputFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    //Load tasks from the file
+    /**
+     * Returns a TaskList object that can be manipulated to get the tasks.
+     * @return a list of tasks objects.
+     * @throws IOException when the file cannot be found.
+     */
+    public TaskList load() throws IOException {
+        TaskList tasks = new TaskList();
+        BufferedReader reader = new BufferedReader(new FileReader(filePath));
+        String line;
+        while((line = reader.readLine()) != null) {
```

Comment:
Should have a white space between `while` and `((line = reader.readLine()) != null)`

Reply:
Done!